core/new-gui/package.json

```diff
@@ -63,7 +63,6 @@
     "html2canvas": "1.4.1",
     "jointjs": "3.5.4",
     "jquery": "3.6.0",
```

Comment:
can we remove "jquery" yet?

Reply:
Minimap is still using jQuery, We can remove it after #2301.